complete-application/src/index.ts

```diff
@@ -22,6 +22,9 @@ const userDetails = 'userDetails'; //Non Http-Only with user info (not trusted)
 const client = new FusionAuthClient('noapikeyneeded', fusionAuthURL);
```

Comment:
I didn't catch this earlier, but the "Logout" button is not replacing the variable correctly and is sticking to `{{logoutUrl}}`